docs/data-sources/rulesets.md

```diff
@@ -44,7 +44,7 @@ Optional:
 - `id` (String) The ID of the Ruleset to target.
 - `kind` (String) Type of Ruleset to create. Available values: `custom`, `managed`, `root`, `schema`, `zone`.
 - `name` (String) Name of the ruleset.
-- `phase` (String) Point in the request/response lifecycle where the ruleset will be created. Available values: `ddos_l4`, `ddos_l7`, `http_custom_errors`, `http_log_custom_fields`, `http_request_cache_settings`, `http_request_firewall_custom`, `http_request_firewall_managed`, `http_request_late_transform`, `http_request_late_transform_managed`, `http_request_main`, `http_request_origin`, `http_request_dynamic_redirect`, `http_request_redirect`, `http_request_sanitize`, `http_request_transform`, `http_response_firewall_managed`, `http_response_headers_transform`, `http_response_headers_transform_managed`, `magic_transit`, `http_ratelimit`, `http_request_sbfm`, `http_config_settings`.
+- `phase` (String) Point in the request/response lifecycle where the ruleset will be created. Available values: `ddos_l4`, `ddos_l7`, `http_custom_errors`, `http_log_custom_fields`, `http_request_cache_settings`, `http_request_firewall_custom`, `http_request_firewall_managed`, `http_request_late_transform`, `http_request_late_transform_managed`, `http_request_main`, `http_request_origin`, `http_request_dynamic_redirect`, `http_request_redirect`, `http_request_sanitize`, `http_request_transform`, `http_response_firewall_managed`, `http_response_headers_transform`, `http_response_headers_transform_managed`, `magic_transit`, `http_ratelimit`, `http_config_settings`. `http_request_sbfm` is also available, but will be deprecated sometime in the future soon.
```

Comment:
the files in `docs/` are automatically generated based on the schema descriptions - eg https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/framework/service/rulesets/schema.go#L102. in the description we use a helper that references `cloudflare.RulesetPhaseValues()` which pulls all the types from `cloudflare-go` - https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L91-L117.

to get this doing what you want, you'll need to:

- remove https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L114 from cloudflare-go
- update the schema description value to include your notice at the end.